### ISSUE TITLE
Mandatory fields are not marked

### ISSUE THREAD

**Issue by matthias-ronge** [kitodo/kitodo-production#3450]
The metadata entry does not show which of the fields are mandatory.

![ Screenshot 1](https://user-images.githubusercontent.com/3040657/79357408-45913480-7f40-11ea-970a-ffd8f0708c66.jpg)

Only after clicking "Save" will they be shown in red.

![Screenshot 2](https://user-images.githubusercontent.com/3040657/79357423-4a55e880-7f40-11ea-9a8f-22593d760954.jpg)

For comparison, in version 2, these were marked with an *.

**Goal:** An asterisk (*) is displayed behind the mandatory metadata fields.

**Comment by andre-hohmann** [comment]
@matthias-ronge : Maybe this related to the following issue?  
- #3403

**Comment by matthias-ronge** [comment]
You wrote  #3403 − you should decide if this is a duplicate of what you meant.

**Comment by andre-hohmann** [comment]
It seems, as if it is very similar, but not the same problem: 
- #4949

But it seems to be a good example that we some labels to connect similar issues. 
Here, it is a good point to mark mandatory fields with an asterix (*) to express for the users, that the field is a mandatory one. 
Still, it needs needs to be checked during the creation of the process, that mandatory fields have to contain a value. 
Both demands should be combined.
